sdk/core/amqp-common/.gitignore

```diff
@@ -0,0 +1,66 @@
+# Logs
+logs
+*.log
+
+# Runtime data
+pids
+*.pid
+*.seed
+
+# Directory for instrumented libs generated by jscoverage/JSCover
+lib-cov
+
+# Coverage directory used by tools like istanbul
+coverage
+
+# Grunt intermediate storage (http://gruntjs.com/creating-plugins#storing-task-files)
+.grunt
+
+# Compiled binary addons (http://nodejs.org/api/addons.html)
+build/Release
+
+# Dependency directory
+# Commenting this out is preferred by some people, see
+# https://www.npmjs.org/doc/misc/npm-faq.html#should-i-check-my-node_modules-folder-into-git-
+node_modules
+
+# Users Environment Variables
+.lock-wscript
+
+# WebStorm files
+.idea
+
+# WebStorm
+.idea
+
+# coverage #
+.nyc_output
+coverage
+
+# Node #
+test-results.xml
+
+# Mac OS #
+.DS_Store
+.DS_Store?
+
+# Nuget packages and corresponding codegen sources
+.nuget
+packages
+
+# Visual Studio #
+*.suo
+*.user
+.ntvs_analysis.dat
+obj/*
+.env
+typings/
+dist/
+dist-esm/
+browser/
+output/
+client/examples/js/lib/
+processor/examples/js/lib/
```

Comment:
Are these client and processor folder paths for the eventhub? If yes, I think these too should get updated to reflect the current path correctly

Reply:
This seems to be an IDE specific setting and not used by the package as such, hence removed it